vertical-pod-autoscaler/pkg/recommender/checkpoint/checkpoint_writer.go

```diff
@@ -119,6 +119,17 @@ func (writer *checkpointWriter) StoreCheckpoints(ctx context.Context, now time.T
 			vpaCheckpoint.ObjectMeta.Annotations[RSSBinaryDecayingHistogramNumBuckets] = fmt.Sprintf("%d", containerCheckpoint.RSSHistogram.NumBuckets)
 			vpaCheckpoint.ObjectMeta.Annotations[JVMHeapCommittedBinaryDecayingHistogramNumBuckets] = fmt.Sprintf("%d", containerCheckpoint.JVMHeapCommittedHistogram.NumBuckets)
 
+			// Annotate the checkpoint with the new timestamp fields.
+			if !aggregatedContainerState.LastSampleStart.IsZero() {
+				vpaCheckpoint.ObjectMeta.Annotations["cpu_last_updated"] = aggregatedContainerState.LastSampleStart.String()
```

Comment:
What formatting is used by Time.String()? Is it hardcoded in Time implementation and always the same e.g. is it possible that it might change due to VM locale? The format for serialization and deserialization has to match so let's use an explicit format for serialization (we already use explicit deserialization format).